scripts/pre_commit_linter.py

```diff
@@ -1140,7 +1140,7 @@ def _lint_py_files(
 
 
 class LintChecksManager(object):
-    """Manages all the linting functions.
+    """Manages all the common linting functions.
```

Comment:
Also mention here that this class isn't intended to be used directly.

Reply:
Done